proxy/src/main/java/net/md_5/bungee/BungeeSecurityManager.java

```diff
@@ -18,30 +18,25 @@
 
     private void checkRestricted(String text)
     {
-        Class[] context = getClassContext();
-        for ( int i = 2; i < context.length; i++ )
-        {
-            ClassLoader loader = context[i].getClassLoader();
+        ClassLoader loader = getClassContext()[0].getClassLoader();
```

Comment:
This is wrong anyway

Reply:
why?

Reply:
should i use getClass().getClassLoader() ?

Reply:
Compare the code before and after

Reply:
oh its 2 haha i'm sorry